html/modules/custom/hr_paragraphs/hr_paragraphs.module

```diff
@@ -528,14 +538,33 @@ function hr_paragraphs_preprocess_paragraph__iframe(&$variables) {
     $ratio = 'ratio-16-9';
   }
 
-  $variables['content']['embed'] = [
-    '#theme' => 'hr_paragraphs_iframe',
-    '#embed_url' => $embed_url,
-    '#width' => $width,
-    '#height' => $height,
-    '#ratio' => $ratio,
-    '#weight' => 99,
-  ];
+  if (strpos($embed_url, 'https://public.tableau.com/views') === 0) {
+    $height = '900px';
```

Comment:
always?

Reply:
Tableau sets it on their embed code, but it's between 800 and 1200px, since I don't know the height from the embed url I have to guess ;-)

Reply:
I'll see if I can find is somewhere

Reply:
I think it's fine for our layout especially to cap it. I just wanted to be sure that (1) it actually was hardcoding, and if true (2) you meant for that.

Reply:
Width is enforced, height is set once the tableau is loaded